resources/views/layout/_footer.blade.php

```diff
@@ -57,7 +57,7 @@
                         <i class="fa fa-user-secret text-primary" aria-hidden="true"></i>
                         Privacy policy<br>
                     </a>
-                    <a href="/code-of-conduct">
+                    <a href="/public/pdf/Code_of_Conduct.pdf">
```

Comment:
This should be,

```suggestion
                    <a href="/pdf/Code_of_Conduct.pdf">
```

the files and folders in the `public` folder are exposed by Laravel as if they are in the root of the website.